### PR TITLE
fix #13 remove Twig.logic.type.include from render tag extension definition

### LINKED ISSUE #13: TwigException: Unable to parse 'include...' since latest update
Getting `TwigException: Unable to parse 'include '@page-link' with link'` and similar whenever I try to `include` another template, since the latest round of updates.

Potential answer seems to be [here](https://github.com/kss-node/kss-node/issues/467#issuecomment-406844915), to do with new namespacing rules in twig.js, but I've tried the solutions in that comment thread and can't figure it out.

### PR DESCRIPTION
**Pull request by therealpecus**
fixes #13
it seems that using twig's internal types overrides the language tags

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@frctl/twig",
-  "version": "1.0.0-beta.4",
+  "version": "1.0.0-beta.5",
```
Comment (Chapabu) [inline package.json]: I'll check this out as soon as I get a chance, but in the meantime would you mind getting rid of the version bump please? We'll bump the version if/when we make a release so we don't miss a release by mistake.

Comment (Chapabu) [inline package.json]: I'll check this out as soon as I get a chance, but in the meantime would you mind getting rid of the version bump please? We'll bump the version if/when we make a release so we don't miss a release by mistake.

**Review by Chapabu — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Chapabu) [reply package.json]: Also, I forgot to say thank you - so...thank you for this :)

Comment (therealpecus) [reply package.json]: Hi Matt, no worries, version reverted. You might want to squish commit when merging.

**Comment by baerkins** [comment]
I can confirm that this fix solved the `include` issue for me. Any chance of getting this merged? :)

**Comment by Chapabu** [comment]
Thanks for this @therealpecus :)

**Pull request merged**